app/javascript/App/containers/newsfeed/sagas.js

```diff
@@ -1,114 +1,122 @@
-import { takeLatest, select, put, fork } from 'redux-saga/effects'
-import { delay } from 'redux-saga'
-import { createEntitySagas, createFilterSagas } from '../../lib/redux'
-import selectors from './selectors'
-import actions from './actions'
-import { namespace } from './constants'
-import { buildFilterObject } from '../../lib/stateHelpers'
+import { takeLatest, select, put, fork } from "redux-saga/effects";
+import { delay } from "redux-saga";
+import { createEntitySagas, createFilterSagas } from "../../lib/redux";
+import selectors from "./selectors";
+import actions from "./actions";
+import { namespace } from "./constants";
+import { buildFilterObject } from "../../lib/stateHelpers";
 
-const entitySagas = createEntitySagas(namespace)
-const filterSagas = createFilterSagas(namespace)
+const entitySagas = createEntitySagas(namespace);
+const filterSagas = createFilterSagas(namespace);
 
 export default function* watcher() {
-  yield takeLatest('ON_FILTER_INITIALIZE', fetchCoins)
-  yield takeLatest('ON_FILTER_INITIALIZE', pollNewsItems)
-  yield takeLatest('SET_ENTITY_LIST', onSetCoinList)
-  yield takeLatest('SET_ACTIVE_ENTITY', onSetActiveCoin)
-  yield takeLatest('ON_FILTER_CHANGE', onFilterChange)
-  yield takeLatest('TOGGLE_UI', onWatchingOnly)
-  yield takeLatest('FETCH_MORE_NEWS_FEED', onScrollingToBottom)
-  yield fork(filterSagas)
-  yield fork(entitySagas)
+  yield takeLatest("ON_FILTER_INITIALIZE", fetchCoins);
```

Comment:
be sure to get the project setup on linting...not sure why you switched to double quotes.